RELEASE.md

```diff
@@ -6,23 +6,27 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 1. Merge all PRs intended for the release into the `master` branch
 1. Checkout and update the master branch and ensure all tests are passing:
-	* `git checkout master`
-	* `git pull`
+    * `git checkout master`
+    * `git pull`
+    * install and update necessary dependencies:
+        * go get -v -u honnef.co/go/tools/cmd/staticcheck
+        * go get -v -u golang.org/x/lint/golint
+        * go get -v -u github.com/securego/gosec/cmd/gosec
```

Comment:
As you mention, some options here:
- Add a "prerequisites" section the development section of the `README.md` which details these and we link to here.
- Alternatively we could have a Make target `install-dependencies` or similar would checks for existence of each binary and installs if not present and add to `make all`.

Reply:
I went with the second option, and replaced the installation steps in the GHA workflow to `make dependencies` instead. Let me know what you think.